### PR TITLE
Add csi-rclone as a sub chart

### PR DESCRIPTION
**Pull request by qrtt1**
- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**

Make csi-rclone install with PrimeHub

### CODE REVIEW

**Comment by popcornylu** [comment]
Can we put the rlcone specific setting in a top-level value? like.
```
rclone:
   kubeletPath:
   image:
   resources:
```

**Comment by popcornylu** [comment]
Please provide images in `images.txt`

File: chart/values.yaml [hunk 1]
```diff
@@ -586,6 +586,8 @@ store:
     enabled: true
   logPersistence:
     enabled: true
+  microk8s:
```
Comment (jackklpan) [inline chart/values.yaml:589]: should be deleted

**Comment by jackklpan** [comment]
LGTM

**Pull request merged**